crates/goose-cli/src/session/output.rs

```diff
@@ -491,6 +491,23 @@ fn get_tool_params_max_length() -> usize {
         .unwrap_or(40)
 }
 
+fn print_value(value: &Value, debug: bool) {
+    let formatted = match value {
+        Value::String(s) => {
+            if !debug && s.len() > get_tool_params_max_length() {
```

Comment:
this shows up as:

<img width="285" height="85" alt="image" src="https://github.com/user-attachments/assets/5fbf830f-0dac-4906-868b-82d04f4b7715" />

is that intended when not debug?

Reply:
I don't know - I didn't really touch this code, the new linter rules force me to break up this function

Reply:
ah - fair enough, yes refactoring makes reviewing diffs very difficult when it breaks up things (wish the diff was a bit smarter, and could point out the code previously existed - if you search for it you can see but easy to miss). Yeah makes sense, just odd I never saw this before.